userspace/libsinsp/sinsp_ppm_sc.cpp

```diff
@@ -164,56 +164,27 @@ std::unordered_set<uint32_t> sinsp::enforce_simple_ppm_sc_set(std::unordered_set
 	return ppm_sc_set;
 }
 
+/// TODO: We need to remove it or use the information we found in the event table
```

Comment:
We could also remove these APIs, otherwise, we need to wait until the `event_table` will allows us to retrieve the `ppm_sc` associate with the event.

Reply:
that is a requirement to solve other issues like https://github.com/falcosecurity/falco/issues/2326

Reply:
Well, couldn't we find all the events with the desired category, and then ask scap to give us the ppm_sc given the event set?

Reply:
Yeah for sure we can do that here. BTW I think we need a more general solution, from our drivers we always receive event types and we should be able to link an event with its "source" (that could be a PPM_SC or a PPM_TP). If it is a syscall event we can easily loop all the syscall tables to find the ppm_sc corresponding to the event (even if very ugly), but for tracepoints, we don't have any way to do that, this is why I was proposing to add the "source" of the event in the event table

Reply:
BTW as a first step we can loop in these methods as you suggested :)